tekton/pipelines/install.yaml

```diff
@@ -516,16 +563,25 @@ spec:
     - name: db2
       params:
         - name: db2_meta_storage_class
-          value: $(params.storage_class_rwx)
+          value: $(params.storage_class_rwo)
         - name: db2_data_storage_class
           value: $(params.storage_class_rwo)
         - name: db2_backup_storage_class
-          value: $(params.storage_class_rwx)
+          value: $(params.storage_class_rwo)
```

Comment:
Rever back to rwx; problem to have it set might be before you even reach install.yml

Reply:
Done - reverted to rwx. I am investigating how this is getting populated in the first place